providers/pg_database.rb

```diff
@@ -26,6 +27,7 @@ def createdb_command
   cmd << "--template #{new_resource.template}"
   cmd << "--encoding #{new_resource.encoding}"
   cmd << "--owner #{new_resource.owner}" if new_resource.owner
+  cmd << "#{connection_string}" unless node[project_name]['postgres']['enable']
```

Comment:
These all need to be `node[project_name]['postgresql']` not `node[project_name]['postgres']`.